megamek/build.gradle

```diff
@@ -394,7 +402,7 @@ task javadocJar(type: Jar) {
 publishing {
     publications {
         publishMMLibrary(MavenPublication) {
-            artifactId = "megamek${mmBranchTag}"
+            artifactId = "megamek${mmBranchTag.replaceAll('[^A-Za-z0-9_\\-.]','-')}"
```

Comment:
Some characters (notably `/`) are allowed for branch names but not for maven artifact names (I changed this in all three projects - no need to check).